README.md

```diff
@@ -68,6 +178,7 @@ The `python-wechaty-plugin-contrib` project was created.
 
 - @wj-Mcat - [wj-Mcat](https://github.com/wj-Mcat), nlp researcher
 - @huan - [Huan LI](https://github.com/huan) ([李卓桓](http://linkedin.com/in/zixia)), Tencent TVP of Chatbot
+- @markoxu - [@markoxu](https://github.com/markoxu), Software Development Engineer at Intel Corporation.
```

Comment:
`markoxu` is the contributor not the maintainer.  Please remove it from the docs.